hardware/tests/test_benchmark_cpu.py

```diff
@@ -99,16 +99,3 @@ def test_run_sysbench_cpu_bytes(self, mock_popen, mock_cpu_socket,
                                            '--cpu-max-prime=15000 run',
                                            shell=True, stdout=subprocess.PIPE)
         self.assertEqual([('cpu', 'logical', 'loops_per_sec', '123')], hw_data)
-
-    def test_run_sysbench_cpu_text(self, mock_popen, mock_cpu_socket,
```

Comment:
Why are you dropping this ?

Reply:
because the output of commands in Python 3.x is bytes, not pure text, so we don't need this, we already have the correspondent bytes test